### PR TITLE
Models and Source term draft

### LINKED ISSUE #25: Models and Source Terms

### PR DESCRIPTION
**Pull request by nandu90**
Will close #25 

Ready for review

### CODE REVIEW

File: doc/source/glossary.rst [hunk 1]
```diff
@@ -95,3 +95,12 @@ Glossary
     UIUC
       The University of Illinois at Urbana-Champaign
 
+    TKE
```
Comment (JezSw) [inline doc/source/glossary.rst:98]: When content complete these should be placed in the correct alphabetical order with the other terms

File: doc/source/references.rst [hunk 2]
```diff
@@ -25,3 +25,17 @@ References
     *Flow Measurement and Instrumentation* **52** 101-114 (2016)
 
 .. [Wilcox] Wilcox, D.C. "Turbulence Modeling for CFD," *DCW Industries* (2006)
+
+.. [Tombo2024] Tomboulides, Ananias, Nadish Saini, Dillon Shaver, Aleksandr Obabko, Haomin Yuan, Elia Merzari, and Paul Fischer. "A Robust Spectral Element Implementation of the K–Τ Rans Model in Nek5000/Nekrs." *Available at SSRN 4785448.* (2024)
```
Comment (JezSw) [inline doc/source/references.rst:29]: Similarly can these be put alphabetically with the rest of the references

**Comment by JezSw** [comment]
A couple of comments and edits formatting wise but needs a content review.

**Comment by JezSw** [comment]
TODO

- [ ] Content additions
- [ ] Test examples run
- [ ] Add links to/name actual examples as appropriate

File: doc/source/glossary.rst (hunk 1 above)
Comment (JezSw) [reply doc/source/glossary.rst:101]: Looks fixed

File: doc/source/references.rst (hunk 2 above)
Comment (JezSw) [reply doc/source/references.rst]: Looks fixed

**Comment by JezSw** [comment]
@nandu90 Rupert mentioned the following to me referring to the formula @ https://github.com/JezSw/nekRS/blob/e2bd11a6334ab5d748bff27f3ca5f9e6b275ba57/doc/source/theory.rst?plain=1#L38. Could you check this makes sense?

I think viscosity (mu) is missing from the no-stress formulation:
I think it should be nabla.tau = mu laplacian u - maybe check with someone though in case I'm just missing something.

**Comment by nandu90** [comment]
> @nandu90 Rupert mentioned the following to me referring to the formula @
> 
> https://github.com/JezSw/nekRS/blob/e2bd11a6334ab5d748bff27f3ca5f9e6b275ba57/doc/source/theory.rst?plain=1#L38
> 
> . Could you check this makes sense?
> I think viscosity (mu) is missing from the no-stress formulation: I think it should be nabla.tau = mu laplacian u - maybe check with someone though in case I'm just missing something.

Thanks for pointing this out. The theory section is updated with correct and concise definition of stress tensor. Note that the most of the theory section is consistent with Nek5000 documentation (https://nek5000.github.io/NekDoc/theory.html#)

**Pull request merged**